CHANGELOG.md

```diff
@@ -5,6 +5,10 @@ versioning. Rather than a static releases, this repository contains of a number
 of regression tests that are each semi-independent.  This CHANGELOG file should be used
 to document pull requests to this repository.
 
+## 2024-10-16 ([#95](https://github.com/nasa/harmony-regression-tests/pull/95))
+
+- TODO: Update this with the changes added for the subet-band-name service
```

Comment:
this is for you @cbrownstevenson

Reply:
@cbrownstevenson - Just noting that this to-do is something that is left to do.